client/src/pages/Profile.js

```diff
@@ -349,12 +349,19 @@ const Profile = ({
 
   const handlePostLike = async (postId, liked, create) => {
     sessionStorage.removeItem("likePost");
+    let currentUser;
+    try {
+      const res = await axios.get("/api/users/current");
+      currentUser = res.data.id;
+    } catch (error) {
+      console.log({ error });
+    }
```

Comment:
I don't think we will need an API call to get the user id here.
Why is the `user._id` not working in the first place in this specific case, I think there might be a much simpler solution that doesn't require an extra API call.

Reply:
Good catch @Zaydme .

The issue was they were trying to use the same user as the profile so of course it was forbidden:

![image](https://user-images.githubusercontent.com/10546720/98679531-2ddf2a80-232e-11eb-83ce-8ecae64b15a1.png)

However, we are completely changing `handlePostLike` with #1767 and it is working there as expected. Thanks for the contribution @mathigayathri2001, but I don't think it's worth it to fix these issues and re-review. I would propose we close this & aim to get #1767 working, reviewed & merged.

Reply:
Thanks for the feedback.